x-pack/test/functional/services/ml/single_metric_viewer.ts

```diff
@@ -117,5 +117,89 @@ export function MachineLearningSingleMetricViewerProvider({ getService }: FtrPro
       await testSubjects.click('mlAnomalyResultsViewSelectorExplorer');
       await testSubjects.existOrFail('mlPageAnomalyExplorer');
     },
+
+    async openConfigForControl(entityFieldName: string) {
+      const isPopoverOpened = await testSubjects.exists(
+        `mlSingleMetricViewerEntitySelectionConfigPopover_${entityFieldName}`
+      );
+
+      if (isPopoverOpened) {
+        return;
+      }
+
+      await testSubjects.click(
+        `mlSingleMetricViewerEntitySelectionConfigButton_${entityFieldName}`
+      );
+      await testSubjects.existOrFail(
+        `mlSingleMetricViewerEntitySelectionConfigPopover_${entityFieldName}`
+      );
+    },
+
+    // TODO move to some common service for EUI components
+    async assertRadioGroupValue(testSubject: string, expectedValue: string) {
```

Comment:
We could move the radio group helper functions to the [ML common UI](https://github.com/elastic/kibana/blob/master/x-pack/test/functional/services/ml/common_ui.ts) service for starters, so we can re-use it in our tests. But this could also be done in a follow-up PR.

Reply:
Refactored in https://github.com/elastic/kibana/pull/81510/commits/ca7c2fc0382dd05b98395d6264db8f2a43240f21